frappe/core/doctype/doctype/doctype.py

```diff
@@ -1266,6 +1266,8 @@ def check_link_table_options(docname, d):
 			return
 
 		if d.fieldtype in ("Link", *table_fields):
+			d.fetch_from = None
```

Comment:
This makes no sense. It's a valid use case! This is a bug in form builder or some other code.

It's used in 170 fields!

![image](https://github.com/frappe/frappe/assets/9079960/a1093fd3-29a4-4461-a88e-580caab72ab2)

Reply:
Closing